test/test_genericprocess_testcase.py

```diff
@@ -0,0 +1,47 @@
+# pylint: disable=missing-docstring
```

Comment:
I'd prefer if this was located under test/tests.

Reply:
Changed in latest fixup (to be squashed).